go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/ghodss/yaml v1.0.0 // indirect
 	github.com/go-openapi/spec v0.19.2 // indirect
 	github.com/gogo/googleapis v1.2.0
-	github.com/gogo/protobuf v1.2.2-0.20190415061611-67e450fba694 // indirect
+	github.com/gogo/protobuf v1.2.2-0.20190415061611-67e450fba694
```

Comment:
Is it right that this PR also has changes to go.mod and go.sum?

Reply:
Yes, go modules updates those files automatically.